src/main/java/com/github/onsdigital/logging/v2/event/Error.java

```diff
@@ -15,35 +16,20 @@
     private Map<String, Object> data;
 
     public Error(Throwable t) {
-        this.message = t.getMessage();
-        this.stackTraces = Arrays.asList(t.getStackTrace())
-                .stream()
-                .map(e -> new StackTrace(e))
-                .toArray(size -> new StackTrace[size]);
+        this.message = t.getClass().getName() + ": " + t.getMessage();
```

Comment:
In some functions, we are checking if the throwable exists and others not. Is it possible to call the function without provided the parameter? Is it worth us checking it was given to the function correctly before accessing methods on it?

Reply:
This method is not used outside of this library but I have added a null check to the method which calls this to prevent any potential null pointer ex.